TASVideos/Pages/Account/Register.cshtml

```diff
@@ -49,6 +49,10 @@
 				</div>
 				<span asp-validation-for="Coppa" class="text-danger"></span>
 			</form-group>
+			<form-group>
+				Also make sure you've read the <a href="/SiteRules">Site Rules</a>.<br/>
+				Your data will be stored according to our <a href="/System/PrivacyPolicy">Privacy Policy</a>. Always be mindful of what you share.
```

Comment:
Also unsure about "Always be mindful of what you share." If we link privacy policy in other places it kinda make sense to say it there too, but can't we say it directly on the privacy policy page instead?